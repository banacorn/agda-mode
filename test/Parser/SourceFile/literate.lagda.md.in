This line is ordinary text, which is ignored by Agda.

? {!!}

```
module Whatever where
-- Agda code goes here
?
```

? {!!}
Here is another code block:

```agda
data ℕ : Set where
 zero : {!  !}
 suc  : ℕ → ℕ
```