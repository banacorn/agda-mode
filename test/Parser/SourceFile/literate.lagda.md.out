Hole [0] (112, 113) => (112, 119) "{!   !}"
Hole [1] (196, 202) => (196, 202) "{!  !}"